### ISSUE TITLE
Pre-trained models

### ISSUE THREAD

**Issue by The-ArdaS** [IcarusWizard/PixelwiseRegression#4]
Hi there,
How can I get the pre-trained models for MSRA and ICVL?

Also, in the paper, it is mentioned that 10 epochs was used (with data aug), but the train code has 50 epochs set as default. Which one is correct?

Thanks

**Comment by IcarusWizard** [comment]
Thanks for your question. 

You can find the pre-trained models and the corresponding results at [release](https://github.com/IcarusWizard/PixelwiseRegression/releases/tag/v1.0).

For the experiment details, the default hyper-parameters in this codebase should be correct. But it can be different with the arXiv version since we have made some changes after uploading. You can find the new version at [IEEE TIM](https://ieeexplore.ieee.org/document/9309323).

**Comment by The-ArdaS** [comment]
Thank you very much for the quick response.

Did you use the code from awesome-hand-pose-estimation/evaluation to generate figures 8 and 9?
('mean error (mm) for each joint' and 'the proportion of frame that all joints error are under the given threshold')
If so, which metric type did you use? max-frame or mean-frame?

**Comment by IcarusWizard** [comment]
Yes, I use tools from awesome-hand-pose-estimation to draw the comparison figures. In particular, figures 8 and 9 in the arXiv version are generated by `max-frame`.

**Comment by The-ArdaS** [comment]
Thank you very much.
